systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -604,6 +606,8 @@ private void assertCoMetricResources(String kind, int value) {
 
     @BeforeAll
     void setupEnvironment(ExtensionContext extensionContext) throws Exception {
+        assumeFalse(Environment.isNamespaceRbacScope());
```

Comment:
Why do metrics tests need to be disabled?

Reply:
because this suite works with 3 namespaces and AFAIK the RBAC can work with only one namespace at once so that's why I disabled it.

Reply:
Why does it need multiple namespace? Metrics do not have anything what would be dependent on global scope.

Reply:
but in that suite we deploy 2 Kafka clusters in separate namespace

Reply:
You can add this assume just only to mm test in metrics I think

Reply:
> You can add this assume just only to mm test in metrics I think

No, we can't already discussed offline.